src/runtime/check_transactions.zig

```diff
@@ -98,6 +99,9 @@ pub fn checkFeePayer(
     FeeDetails,
     TransactionRollbacks,
 }) {
+    var zone = tracy.Zone.init(@src(), .{ .name = "checkFeePayer" });
```

Comment:
This is kind of a general question: why is it that we have to specify `name` everywhere, couldn't it get that information from the `@src()`?

Reply:
That's true, I suppose we could. I went with this way because it allowed me to choose when to include more context versus not, e.g. "verifyPoh" is fine as is (we know which function this is), but "clone" isn't (we need to append some context from the type or file).